ui/effects/use-get-livestreams.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+import React from 'react';
+import { BITWAVE_LIVE_API } from 'constants/livestream';
+
+/**
+ * Gets latest livestream info list. Returns null (instead of a blank object)
+ * when there are no active livestreams.
+ *
+ * @param refreshMs
+ * @returns {{livestreamMap: null, loading: boolean}}
+ */
+export default function useGetLivestreams(refreshMs: number) {
+  const [loading, setLoading] = React.useState(true);
+  const [livestreamMap, setLivestreamMap] = React.useState(null);
+
+  React.useEffect(() => {
+    function checkCurrentLivestreams() {
+      fetch(BITWAVE_LIVE_API)
+        .then((res) => res.json())
+        .then((res) => {
+          setLoading(false);
+          if (!res.data) {
+            setLivestreamMap(null);
+            return;
+          }
+
+          const livestreamMap = res.data.reduce((acc, curr) => {
```

Comment:
Maybe this should have some limit so only livestreams with greater than x viewers are shown?